packages/studio/package.json

```diff
@@ -32,6 +32,7 @@
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-modal": "3.16.1",
+    "react-select": "^5.7.4",
```

Comment:
should we regenerate third party notices for this new dependency?

Reply:
Oh hmm would you be okay waiting until we update the third party notices script?